Cargo.toml

```diff
@@ -1,5 +1,6 @@
 [workspace]
-members = ["src/firecracker", "src/jailer"]
+members = ["src/firecracker"]
+exclude = ["src/jailer"]
```

Comment:
intead of excluding jailer from the workspace, we should add `default-members = ["src/firecracker"]`.
This way, we no longer need the additional `--manifest-path` param in the build command.

Reply:
Thanks, I have verified with default-members, it works.